### PR TITLE
Design cleanup

### PR DESCRIPTION
**Pull request by dhogan8**
Some additional design tweaks.

### CODE REVIEW

File: src/components/PaginationPost.module.scss [hunk 1]
```diff
@@ -0,0 +1,80 @@
+@use '../styles/_variables.scss';
```
Comment (dhogan8) [inline src/components/PaginationPost.module.scss:1]: @kevcenteno I created this file as well as `PaginationPost.module.scss`. Both files have a lot of duplicate code so I think we might be able to reduce this, but I'm having trouble figuring out the best way to do this.

Comment (kevcenteno) [reply src/components/PaginationPost.module.scss:1]: You can import `Pagination.module.scss` file in both pagination components and create+use a parent `.paginationPost` class that modifies the default styling for the `paginationPost` component.

Comment (dhogan8) [reply src/components/PaginationPost.module.scss]: Thank you! This is ready for another review.

File: src/templates/Post/Post.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 
 import Layout from '../../components/Layout/Layout';
 import { h1 as H1, hr as Hr, p as P } from '../../components/Mdx';
-import Pagination from '../../components/Pagination';
+import Pagination from '../../components/PaginationPost';
```
Comment (kevcenteno) [inline src/templates/Post/Post.tsx:6]: Small nitpick: This should be `import PaginationPost from '../../components/PaginationPost';` and its usage be updated as well

Comment (dhogan8) [reply src/templates/Post/Post.tsx]: Ah, thank you. Updated.

**Review by kevcenteno — APPROVED** [review]

**Pull request merged**